config/locales/en.yml

```diff
@@ -735,6 +735,7 @@ en:
       content: We're sorry, but something went wrong on our end.
       title: This page is not correct
     '503': The page could not be served due to a temporary server failure.
+    in_reply_not_found: The status you are trying to reply to does not appear to exist.
```

Comment:
This might be the wrong place for this string. Should it not be together with validation strings like "You cannot attach a poll when there's media attachments" etc?

Reply:
I am not sure, those are in `media_attachments.validations`, it's not a namespace that makes sense for this…